docs/book/src/clusteropenstack/configuration.md

```diff
@@ -120,6 +121,10 @@ The flavors for control plane and worker node machines must be exposed as enviro
 
 # Optional Configuration
 
+## Log level
+
+If with `--v=6` the gophercloud client would log it's requests. This would really help during debugging.
```

Comment:
```suggestion
When running CAPO with `--v=6` the gophercloud client logs it's request to the OpenStack  API. This can be really helpful during debugging.
```